cdap-ui/app/cdap/components/ConfigurationGroup/utilities/DynamicPluginFilters.ts

```diff
@@ -195,10 +192,14 @@ export function filterByCondition(
   // condition is not true.
   let propertiesToHide = flatten(
     filters.map((filter) => {
-      let { expression } = filter.condition;
-      expression = expression || getExpressionFromConditionObj(filter);
+      const { expression } = filter.condition;
+      const mapPropertyToShow = (f: IPropertyFilter) =>
+        f.show.map((showConfig) => ({
+          property: showConfig.name,
+          filterName: f.name,
+        }));
```

Comment:
nit: is the prettifier not working here? the formatting doesn't look right..

Reply:
I am not sure. Prettier didn't format anything for me. Is it happening locally when you build?